database/schema.sql

```diff
@@ -500,3 +500,44 @@ update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from
 
 ALTER TABLE dbo.sccv_personal_relationship
   ADD COLUMN is_main_carer varchar(1);
+
+--- deleted person records handling
+ALTER TABLE dbo.dm_persons
+  ADD COLUMN marked_for_deletion boolean;  
+  
+ALTER TABLE dbo.dm_addresses
+  ADD COLUMN marked_for_deletion boolean; 
+  
+ALTER TABLE dbo.dm_telephone_numbers
+  ADD COLUMN marked_for_deletion boolean; 
+  
+ALTER TABLE dbo.sccv_person_other_name
+  ADD COLUMN marked_for_deletion boolean; 
+  
+ALTER TABLE dbo.sccv_allocations_combined
+  ADD COLUMN marked_for_deletion boolean; 
+  
+ALTER TABLE dbo.sccv_warning_note
+  ADD COLUMN marked_for_deletion boolean; 
+  
+ALTER TABLE dbo.sccv_personal_relationship
+  ADD COLUMN marked_for_deletion boolean;   
+  
+CREATE TABLE IF NOT EXISTS dbo.sccv_person_record_to_be_deleted (
+    id bigint GENERATED BY DEFAULT AS IDENTITY NOT NULL CONSTRAINT xpk_person_record_to_be_deleted PRIMARY KEY,
+    fk_id_to_delete bigint,
+    fk_master_id bigint,
```

Comment:
What do you think about just naming these to be more explicit like `fk_person_id_to_delete` and `fk_master_person_id`?

Reply:
that's better. Updated